document-store/src/main/java/org/hypertrace/core/documentstore/query/Query.java

```diff
@@ -125,7 +125,9 @@ public QueryBuilder setSelection(final Selection selection) {
     }
 
     public QueryBuilder setSelections(final List<SelectionSpec> selectionSpecs) {
-      if (CollectionUtils.isNotEmpty(selectionSpecs)) {
+      if (CollectionUtils.isEmpty(selectionSpecs)) {
```

Comment:
Was it crashing? what was the issue?

Reply:
Basically, when we try to set an `emptyList()` as selection list, it was not being set properly.